R/weather.R

```diff
@@ -246,7 +246,7 @@ weather_dl <- function(station_ids,
         if(verbose) message("Formatting station data: ", s)
         w <- weather_format(w = w,
                             preamble = preamble,
-                            stations = stations,
+                            stations = weathercan::stations,
```

Comment:
```suggestion
                            stn = stn,
```

Reply:
The `weather_dl()` function uses `stn` to specify the stations data frame. So if we make this change here, and then below, in the `weather_format()` function change all references to `stations` to `stn`. This should fix the environment conflict while still allowing users to specify their own updated station data frame.